news-bundle/src/Resources/contao/classes/News.php

```diff
@@ -359,79 +356,31 @@ public function getSearchablePages($arrPages, $intRoot=0, $blnIsSitemap=false)
 	 * @param boolean   $blnAbsolute
 	 *
 	 * @return string
+	 *
+	 * @deprecated Deprecated since Contao 4.10. Use the Symfony router instead.
 	 */
 	public static function generateNewsUrl($objItem, $blnAddArchive=false, $blnAbsolute=false)
 	{
-		$strCacheKey = 'id_' . $objItem->id . ($blnAbsolute ? '_absolute' : '');
+		@trigger_error(__METHOD__.' is deprecated since Contao 4.10. Use the Symfony router instead.', E_USER_DEPRECATED);
```

Comment:
Use `trigger_deprecation` here (see #1946).

Reply:
Fixed in 5db43fe2d7f72fb92f3e92804f74419d6fe17ba8